Makefile

```diff
@@ -308,12 +439,9 @@ $(addprefix .build-artefacts/annotated/, $(SRC_JS_FILES) src/TemplateCacheModule
 	${PYTHON_CMD} .build-artefacts/python-venv/bin/pip install "gspread==0.2.2"
 	touch $@
 
-.build-artefacts/ol-requirements-installation.timestamp: .build-artefacts/python-venv
```

Comment:
Might have been needed for the ol3 build in our original file, but now that you're using global closure install, is not needed anymore? I'm just speculating here..